### DISCUSSION TITLE
can't point to 9.0 executable

### DISCUSSION THREAD

**Discussion by thepeteriley** [interactive-data-language/vscode-idl#18] [General]
Hi, 

I'm not sure if this a bug or user error (probably the latter), but after installing IDL 9.0 on a Silicon Mac and configuring the VS code for IDL extension, it pointed to my 8.82 installation (even though from the CLI I get the right path). I edited that in the VS config setting to point to the location of the 9.0 bin dir, but get this error:

Launching IDL!
/Applications/NV5/idl/bin//idl: line 301: /Applications/NV5/bin/bin.darwin.arm64/idl: No such file or directory
/Applications/NV5/idl/bin//idl: line 301: exec: /Applications/NV5/bin/bin.darwin.arm64/idl: cannot execute: No such file or directory

 see that the issue is that "idl/" is missing after the NV5 directory. But it's correctly specified in the IDL executable setting: /Applications/NV5/idl/bin/. 

Is there somewhere else that the extension/VSC is looking that needs to be set? 

Thanks in advance, Pete

**Comment by znorman-harris** [comment]
Hey Pete!


Thanks for reaching out. You actually pointed out a gap in our check for IDL folders which doesn't account for Apple Silicon. I'll get that updated and we should have a new release out in the next week or so.

From looking at your paths, and I'm not an expert at symbolic links, but you might need the full-path to the bin folder. This should be something like the following: `/Applications/NV5/idl90/bin/bin.darwin.arm64`.

Let me know if that doesn't work!


-Zach

**Comment by thepeteriley** [comment]
Hi Zach, 

Thanks - putting in the full path did the trick!

Cheers, Pete

**Comment by znorman-harris** [comment]
Great! I'll close this then since it is resolved. Let us know if there's anything else you need help with!
